Assets/Lighting/Scripts/LightMoveTest.cs

```diff
@@ -0,0 +1,41 @@
+using UnityEngine;
+
+public class LightMoveTest : MonoBehaviour
```

Comment:
This is just here to move the player object around in the test scene to ensure the lights follow the players movement.

Reply:
Okay, This will be removed once we merge the PlayerController with the lighting.